mediapipe/graphs/instantmotiontracking/calculators/matrices_manager_calculator.cc

```diff
@@ -268,7 +270,11 @@ const Matrix3f MatricesManagerCalculator::GenerateUserRotationMatrix(const float
         // with the direction of finger movement from the user (system dependent)
         Eigen::AngleAxisf(-rotation_radians, Eigen::Vector3f::UnitY()) *
         Eigen::AngleAxisf(0.0f, Eigen::Vector3f::UnitZ()) *
-        Eigen::AngleAxisf(0.0f, Eigen::Vector3f::UnitX());
+        // The (M_PI * 0.5) must be added in order to adjust the default
+        // rendering of the object (the object should appear in the upright
+        // orientation upon initial render of the scene - this is entirely
+        // dependent on the construction of the .obj file).
```

Comment:
This means different obj files might not be able to be used here.  This is fine for now, as we can easily "fix" up the obj models themselves while we have so few of them.  But we should add a TODO to handle this the correct way-- the "initial orientation" (and possibly "initial size" and "initial positioning") of the model should be properties of each model/sticker, so the "initial rotation" would be handled separately from the user-rotation calculations, to be combined later into some sort of "overall rotation"